system-int-test/test/cmr/system_int_test/search/generics_test.clj

```diff
@@ -114,4 +114,11 @@
           (testing "Test that generics will work with concept and revision searches."
             (let [results (search-request (format "concepts/%s/%s" concept-id revision-id) "")
                   status (:status results)]
+              (is (= 200 status) "wrong http status")))
+          
+          (testing "Search generic concept by native-id"
+            (let [results (search-request plural-concept-type-name (str "native-id=" native-id))
+                  status (:status results)
+                  body (:body results)]
+              (is (string/includes? body concept-id) "record not found")
```

Comment:
Don't like this kind of verifications, but didn't refactor the existing tests to do it differently in this PR. I hope this refactoring will happen in CMR-8828.